test/parallel_api/algorithm/alg.sorting/sort_by_key.pass.cpp

```diff
@@ -20,6 +20,47 @@
 
 #include "support/utils.h"
 
+template<typename _Policy>
+void
+test_with_std_pol(_Policy&& policy)
+{
+    const int n = 1000000;
+    std::vector<int> keys_buf(n); //keys
+    std::vector<int> vals_buf(n); //values
+
+    // create objects to iterate over buffers
+    auto keys_begin = keys_buf.begin();
+    auto vals_begin = vals_buf.begin();
+
+    auto counting_begin = oneapi::dpl::counting_iterator<int>{0};
+
+    // 1. Initialization of buffers
+    std::transform(policy, counting_begin, counting_begin + n, keys_begin,
+                   [n](int i) { return i%2 + 1; });
```

Comment:
You don't really need capturing `n`, right?

```suggestion
                   [](int i) { return i % 2 + 1; });
```